jsdoc/api_ApiBuild.js.html

```diff
@@ -0,0 +1,278 @@
+<!DOCTYPE html>
```

Comment:
As an alternative to using single quote to mark private functions, we can
mark everything we want to go into the API as @public and use command-line
'access' property to generate docs just for public members.

2016-08-12 13:20 GMT-07:00 Tatiana Goldina <tgoldina@gmail.com>:

> Couple more suggestions:
>
> 1. About structure of JSDoc. I think, it would be preferable to display
> API jsdoc hierarchically around the namespaces:
>
> - firefly
> - firefly.action
> - firefly.action.type
> - firefly.ui
> - firefly.util
> - firefly.util.image
> - firefly.util.table
> - firefly.util.chart
>
> Ideally, we should be able to place functions and components under
> namespaces by using  @memberof <namespace> tag in the *original* function
> doc. I think, we should not rely on single or double star in comment to
> decide what goes into API docs and what does not.
>
> 2. We might want to use jsdoc.json instead of conf.json, then it would be
> more clear why this config file is for.
>
>
> 2016-08-12 10:48 GMT-07:00 Emmanuel Joliet <notifications@github.com>:
>
>> In jsdoc/api_ApiBuild.js.html
>> <https://github.com/Caltech-IPAC/firefly/pull/142#discussion_r74631821>:
>>
>> > @@ -0,0 +1,278 @@
>> > +<!DOCTYPE html>
>>
>> this is a result from generating jsdoc, no need to be in git, only for
>> version tagged or ops in general. jsdocs reflect stable ops version not
>> ongoing unstable 'dev'.
>>
>> —
>> You are receiving this because you commented.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/Caltech-IPAC/firefly/pull/142/files/2edb4558a9fb7578ee73dddcc02eb927dd9dc1cd#r74631821>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/ALU3QBEsdVprN79ZUKSoIzpKKFDWjXWjks5qfLH2gaJpZM4JikmI>
>> .
>>
>
>